### PR TITLE
Removed useless comments and added info in README.

### PR DESCRIPTION
**Pull request by dilipchauhan14**
Removed useless comments and added info in README.md, so approve/ update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # HelpDesk
-Helps you to waste your time.
-//scjnfvjknfd
 
-#editted the file 
+Helps you. :))))
```
Comment (dilipchauhan14) [inline README.md]: Added Info

File: README.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
 # HelpDesk
-Helps you to waste your time.
-//scjnfvjknfd
 
-#editted the file
```
Comment (dilipchauhan14) [inline README.md]: No use of line.

Comment (sammarth-kapse) [inline README.md]: Good Work.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,8 @@
 # HelpDesk
-Helps you to waste your time.
-//scjnfvjknfd
 
-#editted the file 
+Helps you. :))))
 
 #readme file altered
```
Comment (sammarth-kapse) [inline README.md]: fll;vldvldv

**Pull request merged**